### ISSUE TITLE
Using local model hosted remotely

### ISSUE THREAD

**Issue by amd-123** [JudiniLabs/code-gpt-docs#258]
The PC I run VS-code on does not run my LLM. The LLM runs on a separate PC which has a much more powerful GPU.

At the moment I'm using Ollama and it's in server-mode and I connect to it remotely to send queries and receive responses.

How do I configure Code-GPT in this case to use Ollama remotely? When using local LLM's it seems it is only setup for cases where it's running on the same PC, unless I'm missing something?

**Comment by Vokturz** [comment]
Yes, it currently verifies if the Ollama endpoint is accessible on your PC, which [by default ](https://github.com/ollama/ollama/blob/main/docs/faq.md#how-can-i-expose-ollama-on-my-network) uses port 11434.

You could forward port11434 from your PC with the powerful GPU to the other one, that should works.

**Comment by lstep** [comment]
Same question for me, but I'm using LM Studio, hosted on a second machine, running in server mode. I would be nice if the LM Studio (and Ollama) connectors could be configured to query a remote host.

**Comment by Adel242** [comment]
Hello, at the moment we do not have an option to customize the Ollama URL. We will communicate this to the team for discussion.

**Comment by pawlakus** [comment]
See https://github.com/davila7/code-gpt-docs/issues/227#issuecomment-2142803287 for a solution using a simple python3 aiohttp proxy.

**Comment by ChrisColeTech** [comment]
> See https://github.com/davila7/code-gpt-docs/issues/227#issuecomment-2142803287 for a solution using a simple python3 aiohttp proxy.

there is a reason why "link only" replies are typically discouraged.

**Comment by ChrisColeTech** [comment]
How is this not an option? What benefit is it to hide this simple feature from the user?

This was almost a good thing. but here we are a year later
